### DISCUSSION TITLE
colossal ai of stable diffusion outputs a picture with random noise

### DISCUSSION THREAD

**Discussion by yufengyao-lingoace** [hpcaitech/ColossalAI#2440] [Community | General]
When I use your official code, I can train and get the last.ckpt ,but the output  of txt2img.py is a random noise. Who can tell me what is wrong?
![WX20230111-115715@2x](https://user-images.githubusercontent.com/92853025/211725213-45c4fbad-b257-414c-ac1b-3f461f869cc9.png)

**Comment by binmakeswell** [comment]
https://github.com/hpcaitech/ColossalAI/discussions/2460
